docs/docs/domain.mdx

```diff
@@ -343,8 +343,11 @@ information before it is able to predict the weather.
 * **Description**
 
   Slots of type `any` are always ignored during conversations. The property
-  `influence_conversation` cannot be set to `true` for this slot type. If you want to
-  store a custom data structure which should influence the conversation, use a
+  `influence_conversation` cannot be set to `true` for this slot type. 
+  
+  In stories, while annotating slots of type `any` that are in a list or dictionary format, include the set slot value inside single quotes. See example here in [stories](./writing-stories.mdx).
+  
```

Comment:
Slots of type `any` should not be annotated in stories at all, since they are not influencing the conversation

Reply:
I was told differently in the rasa forums - https://forum.rasa.com/t/the-slot-type-any-does-not-seem-to-work-correctly/38674/3

Reply:
@melindaloubser1 I think this is more of a gray area -- they slots can be there and that shouldn't break anything. It's just that -- for the purpose of training -- the slots are useless.

Reply:
Fair enough, then the documentation can rather reflect this since it will apply to other slots with that format in them e.g. adding this sentence to the top of the slots doc (not under any):

>You can include a slot whose value is in dictionary or list format in stories by wrapping the value in single quotes e.g.
```
story: story with custom slot in dictionary format
steps: 
   - slot_was_set:
     - account_data: '{"id":"zpNi3JA2", "fullName":"Ada Lovelace", "accountRenewalDate":"October 3, 2021"}'
```
> Keep in mind you only need to include slot values in your story if the slot has `influence_conversation` set to `true`.

Reply:
I agree with adding it somewhere more general and also with adding the note about `influence_conversation` (though we have to word it carefully because influencing is often turned on by default).

Also, I think what we need even more is adding under the relevant slot type sections (`list`, `any` and custom slots) some concrete examples of slot values (especially lists/dicts) written in stories. Perhaps this would be useful even for simpler slot types, e.g. to clarify if `float` slot values should be quoted or not...

Reply:
I think we should  do that in a separate PR 👍 @ganehviyer do you want to take care of moving the info to the general section and adding the note?